aperture.go

```diff
@@ -316,6 +317,12 @@ func getConfig(configFile string) (*Config, error) {
 		return nil, err
 	}
 
+	// Finally, parse the remaining command line options again to ensure
+	// they take precedence.
+	if _, err := flags.Parse(cfg); err != nil {
+		return nil, err
```

Comment:
I think we need to check the error returned here or where this function is called. If I run `aperture --help` I get the help text twice.

Reply:
yeah. In lnd we do this:

```
loadedConfig, err := lnd.LoadConfig(shutdownInterceptor)
if e, ok := err.(*flags.Error); !ok || e.Type != flags.ErrHelp {
			// Print error if not due to help request.
			_, _ = fmt.Fprintln(os.Stderr, err)
			os.Exit(1)
		}

		// Help was requested, exit normally.
		os.Exit(0)
}
```

Reply:
Ah yeah, I always forget this! 

Gone with a slightly different solution here,wrapping the parser error (then unwrapping), but essentially the same.